### PR TITLE
windowmaker dockapps

### PR DESCRIPTION
**Pull request by bstrik**
two dockapps for windowmaker : a clock and a systemtray

### CODE REVIEW

**Comment by jagajaga** [comment]
Squash commits into one, please.

File: pkgs/applications/window-managers/windowmaker/dockapps/wmcalclock.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+stdenv.mkDerivation {
+  name = "wmCalClock-1.25";
+  src = fetchurl {
+     url = http://www.cs.mun.ca/~gstarkes/wmaker/dockapps/files/wmCalClock-1.25.tar.gz;
+     sha256 = "4b42b55bb7c1d7c58b5ee1f0058c683d3e4f3e3380d3a69c54a50b983c7c1b3f";
+  };
+
+  buildInputs = [ pkgconfig libX11 libXpm libXext ];
+
+  postUnpack = "sourceRoot=\${sourceRoot}/Src";
```
Comment (jagajaga) [inline pkgs/applications/window-managers/windowmaker/dockapps/wmcalclock.nix]: Why backslash here?

**Comment by jagajaga** [comment]
Check travis.
`error: undefined variable ‘libXfixes’`

**Comment by jagajaga** [comment]
Why did you close? You can do changes, rebase and force push.

**Comment by bstrik** [comment]
I'm just starting with git and was following this [guide](https://nixos.org/wiki/Contributing_to_nixpkgs)
I will have to study git a bit so at the moment it was easier for me to start all over.
When I have more experience I will certainly follow your advice.
Thank you for beying so kind to a newbie.
I have filed a new pull request [6443](https://github.com/NixOS/nixpkgs/pull/6443) so please close this one.

**Pull request closed without merging**